src/app/pages/poll/poll.component.ts

```diff
@@ -91,12 +99,15 @@ export class PollComponent {
    * @param url Customized input url
    */
   onSlugChange(url: string): void {
-    if (!url || url.length === 0) return;
+    if (!url || url.length === 0) {
```

Comment:
same as `!url`, see [this TypeScript playground](https://www.typescriptlang.org/play?#code/MYewdgzgLgBArgJwDYwLwwEQYNwFgBQAlgGYwAUAhIkgJQwDeBMzMokISApgHRIgDmZAORRO0ITTz4AvkA)